crypto/multikey/kind_test.go

```diff
@@ -156,15 +158,108 @@ func TestKeyKindFromSignature(t *testing.T) {
 				t.Errorf("KeyKindFromSignature() err = %v, want %v", err, tt.wantErr)
 			}
 
-			if !assert.Equal(key, tt.wantKey) {
+			if !assertion.Equal(key, tt.wantKey) {
```

Comment:
If you change to `if !assert.Equal(t, key, tt.wantKey) {` this will give more information about what test case failed.